config/cache.php

```diff
@@ -78,4 +78,17 @@
 
     'prefix' => 'october',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Disable Request Cache
+    |--------------------------------------------------------------------------
+    |
+    | The request cache remembers cache retrievals from your cache store
```

Comment:
Change `remembers` to `stores`, `your` to `the`, and replace the last line with `Set to true to disable this in memory request cache` please. Thanks!

Reply:
Done 👍